content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -188,7 +188,7 @@ at [ovn-kubernetes](https://github.com/openvswitch/ovn-kubernetes).
 
 ### Weave Net from Weaveworks
 
-[Weave Net](https://www.weave.works/products/weave-net/) is a
+[Weave Net](https://www.weave.works/oss/net/) is a
 resilient and simple to use network for Kubernetes and its hosted applications.
```

Comment:
Grammatical nit since it is describing the network and it is a general rule to hyphenate in such cases.
```suggestion
resilient and simple-to-use network for Kubernetes and its hosted applications.
```

Reply:
Ideally, we'd do some cleanup to avoid having an opinion on whether or not Weave Net is simple to use. We aim to keep that kind of opinion out of the tech docs, _especially_ for third party products / projects.